src/main/java/com/connectfour/ConnectFour.java

```diff
@@ -1,18 +1,43 @@
 package com.connectfour;
 
+import java.util.Scanner;
+
 public class ConnectFour {
+
+    private static Display display;
+    private static Scanner scanner;
+    private static PlayerScanner playerScanner;
+    private static Board board;
+    private static Disc disc1;
+
+    private static Player playerOne;
+
+
     public static void main(String[] args){
-        Display display = new Display(System.out);
-        display.println("Connect Four!\n");
+        setUpGame();
 
-        Disc testDisc = new Disc("red", '\u25C9');
-        Disc otherDisc = new Disc("blue",'\u25C9' );
-        Board board = new Board();
+        display.println("Welcome to Connect Four!!\n");
 
         Presenter.printBoard(display, board);
 
-        board.addDisc(testDisc, 0 , 0);
-        board.addDisc(otherDisc, 3 , 4);
+        nextTurn(playerOne);
+
         Presenter.printBoard(display, board);
     }
```

Comment:
You could abstract this out even more.  Main could set up the game and then call a play or run method in another class to start the game.  That class would have the steps to play the game - welcome, present board, take turn etc. You can also write some tests for that class.

Reply:
Yeah I wasn't sure if that was over-coding at this step, since the game isn't looping yet. But I can do that!